vuepress/docs/next/tutorials/devops/plugin-configuration.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebarDepth: 2
+---
+
+# Customize Deployment with Plugin Configuration Profiles
+
+This tutorial describes three methods to utilize configuration profiles in the OperatorConfigMap to specify resource allocation. This provides a simple way to customize the deployment parameters of Entando bundle plugins for improved efficiency.
+
+The three methods to insert Configuration profiles are: 
+1. As [Inline Profile](#method-1-inline-profile),
+2. [Mapped Profile](#method-2-mapped-profile), or 
+3. [Default Profile](#method-3-default-profile)
+
+Currently, the following parameters for the memory and CPU resources are supported for specifying. When you specify a `limit` on a resource, that is the maximum allowed for the container. The `request` for containers in a pod determines which node the pods are placed on, dependent on the availability of that resource. 
```

Comment:
"are supported for specifying" is super awkward... rephrase, e.g. "...resources can be specified"

are pods placed on a node or in a node? can eliminate "of that resource" - resource refers to the node so the object of "availability" is implied and doesn't need to be defined

Reply:
hm well "specified" and "supported" mean different things :/ 

the way the sentence is structured, "that resource" implies the node is the resource. if the resource is the memory/cpu, then those terms need to be introduced - as it is they're absent so the reader wouldn't know "that resource" refers to them